L1Trigger/L1THGCal/src/backend/HGCalHistoClusteringImpl.cc

```diff
@@ -43,18 +43,19 @@ float HGCalHistoClusteringImpl::dR( const l1t::HGCalCluster & clu,
 
 
 std::vector<l1t::HGCalMulticluster> HGCalHistoClusteringImpl::clusterSeedMulticluster(const std::vector<edm::Ptr<l1t::HGCalCluster>> & clustersPtrs,
-                                                                                           const std::vector<std::pair<GlobalPoint, double> > & seeds){
+                                                                                           const std::vector<std::pair<GlobalPoint, double> > & seeds) const
+{
 
     std::map<int,l1t::HGCalMulticluster> mapSeedMulticluster;
     std::vector<l1t::HGCalMulticluster> multiclustersTmp;
 
-    for(auto & clu : clustersPtrs){
+    for(const auto& clu : clustersPtrs){
 
         int z_side = triggerTools_.zside(clu->detId());
 
 
-        double radiusCoefficientA = dr_byLayer_coefficientA_.empty() ? dr_ : dr_byLayer_coefficientA_.at(triggerTools_.layerWithOffset(clu->detId())); // use at() to get the assert, for the moment
-        double radiusCoefficientB = dr_byLayer_coefficientB_.empty() ? 0 : dr_byLayer_coefficientB_.at(triggerTools_.layerWithOffset(clu->detId())); // use at() to get the assert, for the moment
+        double radiusCoefficientA = dr_byLayer_coefficientA_.empty() ? dr_ : dr_byLayer_coefficientA_.at[triggerTools_.layerWithOffset(clu->detId())];
```

Comment:
It doesn't compile. `at()` should be replaced by `[]`

Reply:
right, sorry, fixed.